partner_email_check/tests/test_partner_email_check.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2019 Komit <https://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from unittest.mock import patch
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tests.common import TransactionCase
+from odoo.tools.misc import mute_logger
+
+
+class TestPartnerEmailCheck(TransactionCase):
```

Comment:
pls use `SavepointCase` and use `cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))` to disable tracking globally.